server-1714 | Added note to delete the SA static keys (#6593)
circleci/circleci-docs: jekyll/_cci2/server-3-install-build-services.adoc

    --member "serviceAccount:<GCP_PROJECT_ID>.svc.id.goog[circleci-server/nomad-autoscaler]"
```

> NOTE: If you are switching from static json credentials to Workload Identity, you must delete the keys from GCP in order to avoid the key misuse.

Reviewer: ```suggestion
NOTE: If you are switching from static JSON credentials to Workload Identity, you should delete the keys from GCP.
```
